app/src/main/java/com/example/bill/photostreamapp/DatabaseHelper.java

```diff
@@ -0,0 +1,52 @@
+package com.example.bill.photostreamapp;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.provider.BaseColumns;
+
+/**
+ * Created by bill on 10/4/15.
+ */
+public class DatabaseHelper extends SQLiteOpenHelper {
+    private static final int DATABASE_VERSION = 1;
+    private static final String DATABASE_NAME = "PhotoFeed.db";
+
+    private static final String TEXT_TYPE = " TEXT";
+    private static final String COMMA_SEP = ",";
+    private static final String SQL_CREATE_ENTRIES =
+            "CREATE TABLE " + FeedEntry.TABLE_NAME + " (" +
+            FeedEntry._ID + " INTEGER PRIMARY KEY," +
+            FeedEntry.COLUMN_NAME_URL + TEXT_TYPE +
+            " )";
```

Comment:
Any reason why this is within the helper rather than a database contract? @billmwong